BaseFilter/sections/antiadblock.txt

```diff
@@ -796,8 +796,7 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 ||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$redirect=googlesyndication-adsbygoogle,domain=t2bot.ru,important
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=t2bot.ru
 ! https://github.com/AdguardTeam/AdguardFilters/issues/137972
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=atikrost.com
-@@||googleads.g.doubleclick.net/pagead/id$xmlhttprequest,domain=atikrost.com
+atikrost.com#%#//scriptlet('prevent-eval-if', 'mdp-deblocker')
```

Comment:
Wrong place and no fix for platworms where no scriptlet support or scriptlet may work unstable.

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/6558d2670433fe24ae89775b9a5a3e03deb77281

Reply:
Oh sorry.